run_tests.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
This is definitely a good idea, or migrating the test execution to a custom `setup.py` command